test-e2e/src/lib.rs

```diff
@@ -1,20 +1,50 @@
-use mopro_ffi::{app, WtnsFn};
+extern crate core;
 
-rust_witness::witness!(multiplier2);
-rust_witness::witness!(multiplier2bls);
-rust_witness::witness!(keccak256256test);
-rust_witness::witness!(hashbenchbls);
+// First, configure the Mopro FFI library
+mopro_ffi::app!();
```

Comment:
Ok yes, if we remove the `set_` macros we end up with just this as a macro invocation.